mixins/localize-static-mixin.js

```diff
@@ -0,0 +1,20 @@
+import { LocalizeMixin } from './localize-mixin.js';
+
+export const LocalizeStaticMixin = superclass => class extends LocalizeMixin(superclass) {
+
+	static async getLocalizeResources(langs) {
+		langs.forEach((lang) => {
```

Comment:
This is just the current logic that's used.  I can update the logic to do something similar to what @mpharoah-d2l did for a more granular fallback, but I wanted to open the PR to get feedback on the idea of the static/local mixin approach.